src/pkg/packaging/rpm/templates/changelog

```diff
@@ -0,0 +1,2 @@
+* {DATE} {MAINTAINER_NAME} <{MAINTAINER_EMAIL}> {PACKAGE_NAME} ({PACKAGE_VERSION}-{PACKAGE_REVISION}) unstable; urgency={URGENCY}
```

Comment:
I think the format is non-standard, if not incorrect.

RPM uses this format:  https://fedoraproject.org/wiki/Packaging:Guidelines?rd=Packaging/Guidelines#Changelogs.

It might be a good idea to run `rpmlint` on the generated RPM to find out more issues.